docs/src/administration/cli/api-tokens.md

```diff
@@ -153,19 +167,24 @@ To run a cron only on your production environment, check the environment type as
 
 ```yaml
 crons:
-    backup:
-        spec: 'H 4 * * *'
+    sendemails:
+        spec: '*/7 * * * *'
         commands:
             start: |
                 if [ "$PLATFORM_ENVIRONMENT_TYPE" = production ]; then
-                   platform backup:create --yes --no-wait
+                   cd public && send-pending-emails.sh
                 fi
 ```
 
-{{< note >}}
+## Use the CLI SSH certificate for other commands
+
+After you authenticate the CLI using a `PLATFORMSH_CLI_TOKEN` environment variable,
```

Comment:
:lady_beetle: As I noted above, it's not necessarily fully authenticated. Or I think that's the issue. That it's present and waiting to be used, but not run so no SSH certificate is generated. So I'd adjust the language here as well.